layouts/index.html

```diff
@@ -2,6 +2,19 @@
 <html lang="en">
   <head>
     {{ partial "head.html" . }}
+    
+    <script>
```

Comment:
Should we add this to the `head.html` partial so we can include it on other pages, _as they come down the pipe_. Or is the initialization on `L12-15` custom to the homepage?